content/articles/react-flux-frontend.md

```diff
@@ -0,0 +1,54 @@
+Title: A tour of modern frontend with React and Flux
+Date: 2015-08-18
+Short_summary: React and Flux have swept the frontend in the last year or so. Let's look at it.
+Category: Dev
+Authors: Vincent
+
+
+*Reading time: ~15 minutes.*
+
+TODO: write intro
+
+<!-- PELICAN_END_SUMMARY -->
+
+
+## What are React and Flux?
+React is a library open-sourced by Facebook 2 years ago 
+that focuses on the UI elements.
+It is not a complete framework like Angular or Ember: it only cares about rendering components everytime their data has changed. To keep this fast, React uses a virtual DOM to do diffs rather than looking at the DOM, modifying only the changed bits.
+
```

Comment:
This intro requires too much knowledge (e.g. what's a virtual dom?). I'd either skip it completely, or be more specific (e.g. links, more text)